scripts/node/twig_functions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Install Pattern Lab.
+ */
+const shell = require("shelljs");
+const fs = require('fs');
+const path = {
+  'drupalComposer': '../../../../vendor/drupal-pattern-lab',
+  'drupalNoComposer': './../../vendor/drupal-pattern-lab',
+  'standalone': 'vendor/drupal-pattern-lab'
+}
+
+function vendorPathInstalled() {
```

Comment:
This method could be removed since you're checking that at least one of these things exists implicitly in `vendorDirectoryRoute()`